string[] categoryName
string[] channelName
float32[] activationValue
float32[] channelRelevance